gulpfile.js

```diff
@@ -223,4 +232,9 @@ gulp.task('watch', function() {
 // This task starts google app engine development server.
 // TODO(Barnabas Makonda): check if files are already generated and if so
 // do not build.
-gulp.task('start_devserver', ['build', 'gulpStartGae', 'watch']);
+var isProdMode = argv.prod_env === 'True';
+if (isProdMode) {
+  gulp.task('start_devserver', ['gulpStartGae', 'watch']);
```

Comment:
@seanlip  @MAKOSCAFEE  Should the refactoring of the build process be a separate issue? This PR fixes a major bug, i.e third party build happens after we have copied the generated third party files. Refactoring might take some time.